packages/astro/src/core/add/index.ts

```diff
@@ -60,6 +60,10 @@ export default {
 	preprocess: vitePreprocess(),
 };
 `;
+const LIT_NPMRC_STUB = `\
+# Lit libraries are required to be hoisted for now due to dependency issues
```

Comment:
I wasn't sure whether this was a public-facing user message, but just in case:

```suggestion
# Lit libraries are required to be hoisted due to dependency issues.
```
I would tend to take out things like "for now"... gives us cover!  ;) If this is a public-facing statement, then we usually do make them sentences with punctuation.